javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentClassLoader.java

```diff
@@ -84,7 +84,25 @@ public AgentClassLoader(File javaagentFile) {
    */
   public AgentClassLoader(
       File javaagentFile, String internalJarFileName, boolean isSecurityManagerSupportEnabled) {
-    super(new URL[] {}, getParentClassLoader());
+    this(javaagentFile, internalJarFileName, isSecurityManagerSupportEnabled, null);
+  }
+
+  /**
+   * Construct a new AgentClassLoader with a custom parent ClassLoader.
```

Comment:
something like this, otherwise someone could come along in the future and think it's unnecessary and remove the extra method
```suggestion
   * Construct a new AgentClassLoader with a custom parent ClassLoader.
   * This is used by some 3rd party command-line utilities in order to reuse classes that are bundled as classdata files under `inst/`.
```